### ISSUE TITLE
Object.observe support

### ISSUE THREAD

**Issue by anthonyshort** [ripplejs/ripple#22]
Update ripple/path-observer to use Object.observe if it's available.

**Comment by tarqd** [comment]
Would you be open to using [observe-js](https://github.com/Polymer/observe-js) for this? It already uses Object.observe if available and implements the LCS algorithm that we talked about on the `ripplejs/each` issue

**Comment by anthonyshort** [comment]
Yeah I'd probably just use that library. I haven't had much need for it yet though. I think the only real benefit is a possible slight performance boost.
